src/manager/msWindowManager.ts

```diff
@@ -15,6 +15,7 @@ import { Rectangular } from 'src/types/mod';
 import { assert } from 'src/utils/assert';
 import { AsyncDebounce } from 'src/utils/async';
 import { groupBy } from 'src/utils/group_by';
+import { logAsyncException, mslog } from 'src/utils/log';
```

Comment:
mslog doesn't exist

Reply:
Pushed. I added this because it's easier to import. Using the `Me` object requires me to copy-paste stuff from another file, while using `mslog` just requires me to press tab to automatically add the import in vscode when using autocomplete.

It's functionally identical to Me.log.